aiohttp/helpers.py

```diff
@@ -366,6 +368,10 @@ def _format_T(args):
         return round(args[4])
 
     @staticmethod
+    def _format_Tf(args):
+        return '{:.06f}'.format(args[4])
```

Comment:
python -m timeit '"{:.06f}".format(1.2345)'
1000000 loops, best of 3: 0.804 usec per loop

python -m timeit '"%06f" % 1.2345'
10000000 loops, best of 3: 0.0203 usec per loop